nss-day-cohort-48/Rare-API-Polite-Pirates
# added folders and models

#### Changes Made
1. Added file `comment`, `post_tags`, `categories`, `tags`, `posts`, `users`, `post_reactions`, `reaction` to `Models` directory.

   
#### Steps to Review
1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout jf-folders
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   > Example: When user does BLANK, then BLANK should happen.
4. View code file.
   > Confirm file modifications are present as indicated above.
